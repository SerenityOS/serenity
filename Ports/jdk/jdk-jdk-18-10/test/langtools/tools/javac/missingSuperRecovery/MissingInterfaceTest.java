/*
 * @test /nodynamiccopyright/
 * @bug 8036007
 * @summary javac crashes when encountering an unresolvable interface
 * @build MissingInterfaceTestDep
 * @clean Closeable
 * @compile/fail/ref=MissingInterfaceTest.out -XDrawDiagnostics MissingInterfaceTest.java
 */

public class MissingInterfaceTest {
    void test(MissingInterfaceTestDep s) {
        s.call();
        s.another();
    }
}
