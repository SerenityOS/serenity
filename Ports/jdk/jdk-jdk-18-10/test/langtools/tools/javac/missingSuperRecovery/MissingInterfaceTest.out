MissingInterfaceTest.java:12:10: compiler.err.cant.access: Closeable, (compiler.misc.class.file.not.found: Closeable)
MissingInterfaceTest.java:13:10: compiler.err.cant.resolve.location.args: kindname.method, another, , , (compiler.misc.location.1: kindname.variable, s, MissingInterfaceTestDep)
2 errors
